src/Spec-Core/SpecPragmaCollector.class.st

```diff
@@ -9,6 +9,9 @@ Class {
 	#instVars : [
 		'behavior'
 	],
+	#classInstVars : [
```

Comment:
This variable should not be here.

Reply:
This one should still be removed.

Reply:
still... I didn't touch any of this.
if they are there, is not my fault :)